src/components/dataset/overview/linked-forms.vue

```diff
@@ -85,7 +85,7 @@ export default {
 {
   "en": {
     // Number of form(s) is shown separately above this text
-    "formsConsumeData": "Form consumes data from this Dataset | Forms consume data from this Dataset",
+    "formsConsumeData": "Form consumes Entities from this Entity List | Forms consume Entities from this Entity List",
```

Comment:
Maybe even just "Forms consume this Entity List"?

Reply:
Ooh, how about "Forms use this Entity List"? I think we were doing "consume" to try to mention "Entity" as well as "Dataset". But we probably don't need to do that anymore.